### PR TITLE
Update sensor.py

### PR DESCRIPTION
**Pull request by janvkem**
Add SZ_FILTER_REMAINING

### CODE REVIEW

File: custom_components/ramses_cc/sensor.py [hunk 1]
```diff
@@ -384,6 +384,9 @@ async def async_update(self) -> None:
         DEVICE_UNITS: UnitOfTemperature.CELSIUS,
     },
     SZ_FAN_INFO: {STATE_CLASS: None},
+    SZ_FILTER_REMAINING: {
```
Comment (tomkooij) [inline custom_components/ramses_cc/sensor.py:387]: You need to import this from `ramses_rf.protcol.const`

Comment (janvkem) [reply custom_components/ramses_cc/sensor.py:387]: Yes, just realised that I forgot that. Thanks

**Comment by tomkooij** [comment]
I monkey-patched this (and the ramses_rf PR) into the latest HA running ramses_cc.

It works! The enitity `xxxxx_filter_remaining` now appears in HA en it is read.

image

**Comment by zxdavb** [comment]
see: https://github.com/zxdavb/ramses_cc/commit/fb6a05a32b1ae91d03ae04c1ff7300f7bf7e9e66 [5 days ago](https://github.com/zxdavb/ramses_cc/issues/88#event-11095272895)

**Pull request closed without merging**